cumulusci.yml

```diff
@@ -212,6 +212,8 @@ plans:
                             name: EDA - Account Record Types
                         contact_key_affl_fields:
                             name: EDA - Contact Key Affiliation Fields
+                        case_behavior_record_types:
+                            name: EDA - Case Behavior Record Types
```

Comment:
This seems like a weird name, maybe. Perhaps "EDA - Case Incident Record Type"?

Reply:
Well Incident is one of the record types and I am wondering if we "might" (although not likely) add more unmanaged record types to this file?